articles/architecture-scenarios/implementation/b2b/b2b-provisioning.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Provisioning
+description: Understand user provisioning functionality and considerations for your B2B implementation. 
+toc: true
+topics:
+    - b2b
+    - b2biam
+    - user-migration
+    - custom-db
+    - universal-login
+    - user-profiles
+contentType: concept
+useCase:
+  - user-provisioning
+  - store-user-data
+---
+# Provisioning
+
+<%= include('../../_includes/_provisioning/_introduction.md', { platform: 'b2b' }) %>
+
+## Provisioning organizations
+
+::: panel best practice
+What you need to do when provisioning an organization will depend on how organizations are represented in your system.  This can take some time to step back and consider how users of those organizations will be interacting with your applications.  Please see our [Multiple Organization Architecture documents](https://drive.google.com/a/auth0.com/file/d/1y2G8RNHTBujcCrnMRhp6_phQiRAkZzfF/view?usp=sharing) to get a better feel for how to decide how to have organizations represented in your IAM system.
```

Comment:
We should move this document too the docs repo too.

Reply:
Done